proto/zitadel/management.proto

```diff
@@ -8152,7 +8152,10 @@ message ImportHumanUserResponse {
 }
 
 message AddMachineUserRequest {
-    string user_name = 1 [
+    // optionally set your own id unique for the user.
+    optional string user_id = 1 [(validate.rules).string = {min_len: 1, max_len: 200}];
```

Comment:
Renumbering the attributes is a breaking change, please number the user_id with 4, so that the other attributes are not changed.